tests/flow/test_nul_bytes.py

```
@@ -0,0 +1,114 @@
+import time
+from common import *
```

Ruff (0.16.2)

[error] `from common import *` used; unable to detect undefined names

(F403)